src/constants/transactions.js

```diff
@@ -1,8 +1,10 @@
 export const TRANSACTION = {
   INITIAL_NOTE: 'Welcome to Jibber',
-  EVENT_TYPE: 'NEW_USER',
+  EVENT_TYPE: {
+    NEW_USER: 'NEW_USER'
```

Comment:
make this INTEREST_PAYMENT

Reply:
So instead of NEW_USER, make it INTEREST_PAYMENT. Are we doing to use NEW_USER in the future o just remove it?

Reply:
@Benjamindodgson

Reply:
just remove it.